news/2024/2024-11-15-osulazer-updates-november-15-2024.md

```diff
@@ -0,0 +1,94 @@
+- **Toggles and shortcuts for hitsound sample addition banks.** Seasoned mappers will likely be familiar with the `Shift`/`Alt` + `Q`/`W`/`E`/`R` hitsounding keys already, which now all work on osu!(lazer).
+- **Breaks can now be hidden from the timeline.** Minor visual change for those who find this indicator distracting.
+
+## Gameplay changes
+
+- **Pausing is now instant when hitting the pause key (`Esc`).** This used to be affected by the "hold-to-confirm activation time" setting.
+- **The time it takes to quick retry (`Ctrl` + `R`) has been decreased.** Without arbitrary delays, you'll be back in action as soon as the beatmap loads, i.e. about one second faster than before.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-quick-retry.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Skinning changes
+
+- **New "use relative size" setting for song progress bars.** Best explained with a video:
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-relative-skin-size.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Other changes
+
+- **New osu!mania ruleset selection sample.** Our audio engineer [nekodex](https://osu.ppy.sh/users/102) has been at it again. Take a listen:
+
+Before:
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-ruleset-select-mania-before.mp3">
+    </audio>
+</div>
+
+After:
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-ruleset-select-mania-after.mp3">
+    </audio>
+</div>
+
+- **You can now search chat channels and the friends/online user listing.** Popular and social people will appreciate this one.
+- **New `ranked` and `submitted` date filtering in song select.** This already existed on the website. See the [beatmap search article](/wiki/Beatmap_search) for an extensive list of filters.
+- **Searching mods by acronym will highlight the corresponding mod for selection.** `F1` + mod acronym + `Enter` + `F1`. Now if only there was a way to remember all the new mod acronyms...
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-mod-acronym-search.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **New settings button to reset all beatmap offsets.** Useful when you realise you've set all your offsets wrong.
+- **Local scores are now preserved even when deleting maps.** Quality-of-life change for maintainers of large beatmap libraries.
+- **New context menu option for copying link to an online score.** The same thing exists when right-clicking beatmaps in song select, btw.
```

Comment:
"btw" feels a bit too casual for the writing style of this newspost series

Reply:
```suggestion
- **New context menu option for copying link to an online score.** The same thing exists when right-clicking beatmaps in song select, by the way.
```